src/GeneticSharp.Domain.UnitTests/GeneticAlgorithmTest.cs

```diff
@@ -445,19 +570,19 @@ public void Start_ManyCallsTerminationChanged_NewEvolutions()
 					new FitnessStub() { SupportsParallel = false }, selection, crossover, mutation);
 
 			target.Population.GenerationStrategy = new TrackingGenerationStrategy();
-			target.Termination = new GenerationNumberTermination(100);
+			target.Termination = new GenerationNumberTermination(500);
```

Comment:
Out of 3 builds, 2 were failing because the difference between 25 and 50 generations is really small. As indicated in commit, is just a proposal to avoid multiple builds.